.github/workflows/ci-model-regression.yml

```diff
@@ -17,6 +17,7 @@ concurrency:
 env:
   GKE_ZONE: us-central1
   GCLOUD_VERSION: "318.0.0"
+  DD_PROFILING_ENABLED: true
```

Comment:
The profiler will be enabled for all model regressions workflows, whenever someone runs the workflow. I'd prefer to disable it by default and give an option to explicitly use it e.g. by adding an extra label in a PR.

Reply:
Agreed, as discussed, always running the profiler is wasteful, because the profiler cost additional money.
Also, a developer would know beforehand if they want to profile, in which case he/she can enable it.

Therefore, I will set this to false for now, and create a follow-up PR to introduce the usability feature of enabling via a PR-label.